src/ErrorMessage.tsx

```diff
@@ -25,25 +25,18 @@ const ErrorMessage = <
   }
 
   const { message: messageFromRegister, types } = error;
-  const props = {
-    ...rest,
+  const props = Object.assign({}, rest, {
     children: messageFromRegister || message,
-  };
+  });
```

Comment:
Is this actually less code after bundle? We should do that in hook form as well, every byte counts! ❤️

Reply:
yes it saves some bytes :)

Reply:
That's awesome!